lib/router/index.js

```diff
@@ -76,11 +76,3 @@ export const createRouter = function (...args) {
 
 // Export the actual Router class, which is usually used inside the server
 export const Router = _Router
-
-export function _notifyBuildIdMismatch (nextRoute) {
```

Comment:
That's doable. I think we could improve the BUILD_ID generation later on. I'll do it as a separate PR.